### ISSUE TITLE
the module splits after abbreviations

### ISSUE THREAD

**Issue by Imene1** [andrefs/Lingua-EN-Sentence-Offsets#2]
The bug is in the line 119
## fix where abbreviations exist

```
    foreach (@ABBREVIATIONS){ $unsplit = $1 if $s =~ /\b$_$PAP\s$/is; }
```

$unsplit = $1   should be $unsplit = 1

**Comment by andrefs** [comment]
Thank you for noticing it and taking the time to open an issue. Good catch, I'll fix this as soon as possible.
If you give me an example which highlights this bug I'll also write a test to make sure it won't happen again.

Cheers

**Comment by Imene1** [comment]
Please test the following code: 

use Lingua::EN::Sentence::Offsets qw/get_offsets get_sentences/; 
my $text = 'Is dr. Ahmed in his office ?';
my $offsets = get_offsets($text);  
foreach my $o (@$offsets) {
my $start  = $o->[0];
my $length = $o->[1]-$o->[0];
my $sentence = substr($text,$start,$length);  ## Get a sentence.
print "$start\t$length\t$sentence\n";

Because of the bug it displays: 
0     6    Is dr.
7     21        Ahmed in his office ?

By correcting the bug it will display:
0     28      Is dr. Ahmed in his office ?

**Comment by andrefs** [comment]
I've made a [commit](http://github.com/andrefs/Lingua-EN-Sentence-Offsets/commit/00fd68a8f3c36f9b76ad8f7d0243956faf2847f2) fixing this bug and adding a test case with your example.

While trying to make a new release I've detected a [bug in Dist::Zilla::Plugin::ChangelogFromGit::CPAN::Changes](http://github.com/ioanrogers/Dist-Zilla-Plugin-ChangelogFromGit-CPAN-Changes/issues/12).

I'll upload a new version to CPAN as soon as it is fixed, or sooner if that takes too long. If you run into any other bugs, or if you have any new feature suggestions, please tell me (pull requests are specially welcome!).

Cheers

**Comment by andrefs** [comment]
Just released v0.03 to CPAN, issue has been fixed
